### DISCUSSION TITLE
NFT not been shown in any MarketPlace

### DISCUSSION THREAD

**Discussion by rkoranne0755** [aptos-labs/aptos-developer-discussions#534] [Questions] [answered]
### Describe your question in detail.

i wrote this contract for minting soulbound NFT's, and using the Contract I have minted few NFT on my wallet address on TESTNET.
Now these NFT's are being visible in my petra wallet, but not on any MarketPlace.

Why so?
Contract Address: 0x2a0abf2355b42068acea7a8b5862785bf3656f9225872e2020480ca2eaa3e66c
wallet address: 0x3fb90e96842b5d4035ab1eab45df216b235f3c495777672471523daa199eb188

### What error, if any, are you getting?

The NFT are not showing at any Marketplace.

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by gregnazario** [comment, marked as answer]
Testnet NFTs will only show on Marketplaces that run on Testnet.  I think most Marketplaces only run on Mainnet and that may be the confusion?

**Reply by rkoranne0755** [reply]
So do we have any marketplace that runs for the testnet?
